doc/api/module.md

```diff
@@ -28,7 +28,7 @@ added:
 A list of the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third party or not.
 
-Note: the list doesn't contain [prefix-only modules][] like `node:test`.
+Note: the list also contains [prefix-only modules][] like `node:test`.
 
```

Comment:
```suggestion
```

Reply:
happy to remove it, but my thinking was that this would help call out the change for those already familiar with the API.

Reply:
Let's remove the note and add an entry to the the YAML comment instead